### PR TITLE
ci: add linux deps

### PR DESCRIPTION
**Pull request by clavin**
Linux CI seems to fail due to missing system dependencies required to load chromedriver, which are added in this PR.

### CODE REVIEW

**Review by dsanders11 — CHANGES_REQUESTED** [review]
Blocking merge temporarily just to see what happens with in-flight builds, will unblock in a couple hours.

**Comment by clavin** [comment]
Rebased & squashed after getting successful run: https://app.circleci.com/pipelines/github/electron/chromedriver/369/workflows/f3dccc80-036c-45bf-bba0-c45b726ec8c9/jobs/2970

* Linux deps step ran
* Node tests show that the test passed, not just skipped

**Review by dsanders11 — APPROVED** [review]
This has probably been failing for a while and we're just now noticing because the old smoke test wasn't checking the output for Electron versions 7-29.

LGTM 👍

**Pull request merged**
